templates/types/streaming/fastapi/app/api/routers/chat.py

```diff
@@ -39,23 +39,23 @@ async def chat(
                     MetadataFilter(
                         key="private",
                         value="true",
-                        operator="!=", # type: ignore
+                        operator="!=",  # type: ignore
                     ),
                     MetadataFilter(
                         key="doc_id",
                         value=doc_ids,
-                        operator="in", # type: ignore . This won't work with a simple vector store since it uses the 'any' operator for that.
-                    )
+                        operator="in",  # type: ignore
```

Comment:
@marcusschiesser we still have this issue until the new llama-index-core is released